pipelines/crawler/bndes/constants.py

```
@@ -385,3 +385,117 @@ class constants_administracao_publica(Enum):
+
+    RESOURCE_SHOW_URL = (
+        "https://dadosabertos.bndes.gov.br/api/3/action/resource_show"
+        "?id=0cfe4594-44bf-48a8-a79a-686fc2d0db95"
+    )
+    DOWNLOAD_URL = (
+        "https://dadosabertos.bndes.gov.br/dataset/"
+        "f27e48cd-653b-4bfa-bc4f-08b637793873/resource/"
+        "0cfe4594-44bf-48a8-a79a-686fc2d0db95/download/"
+        "operacoes-exportacao-operacoes-de-exportacao-pos-embarque-bens.csv"
+    )
+    INPUT_PATH = "/tmp/input/br_bndes_operacoes_contratadas/"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/input/br_bndes_operacoes_contratadas/"

(S108)